retrofit_gen/lib/writer/writer.dart

```diff
@@ -22,7 +22,7 @@ class Writer {
 
     sb.write('var req = base.${r.method}');
 
-    if (r.path != null) sb.write('.path("${r.path}")');
+    if (r.path != null) sb.write('.path("\$basePath${r.path}")');
```

Comment:
Use two different calls to `.path` method. That way, Jaguar Resty will take care of missing forward slashes, etc.

Reply:
ho ok !